src/functions/advanced-functions/event.php

```diff
@@ -136,9 +136,7 @@ function tribe_update_event( $postId, $args ) {
 	 * @return bool false if delete failed.
 	 */
 	function tribe_delete_event( $postId, $force_delete = false ) {
-		$success = Tribe__Events__API::deleteEvent( $postId, $force_delete );
-
-		return $success;
+		return Tribe__Events__API::deleteEvent( $postId, $force_delete );
```

Comment:
:no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"

<sub>hash: 2171c9e6bdcee442d9d9e9cd9b91027e</sub>